### ISSUE TITLE
Proposal: Creating has_one association example

### ISSUE THREAD

**Issue by davydovanton** [hanami/hanami.github.io#278]
Now we have only `has_many` association [example](https://github.com/hanami/hanami.github.io/blob/build/source/guides/models/associations.md#has-many). I think it will be good to add `has_one` example too.
WDYT?

/cc @hanami/documentation @jodosha @cllns

**Comment by mereghost** [comment]
@davydovanton it's not implemented right now and even `has_many` is still experimental.

**Comment by AlfonsoUceda** [comment]
@mereghost as far as I know we have implemented all associations isn't it? I think we should create guides for `belongs_to, `has_one` and `many_to_many`.

**Comment by mereghost** [comment]
Yep. `has_one`, `has_many`, `belongs_to` and `has_many :through` (aka many-to-many) working good as far as I made the code bend and stretch itself.
